src/components/MapView.jsx

```diff
@@ -132,6 +135,7 @@ export const MapView = () => {
       const response = await api.tileGetTilesList({
         headers: basicHeaders(),
       });
+      setIsLoading(false);
```

Comment:
Dobrze by było ustawić 'setLoading(false)' za try/catch
Wtedy jak polecić błąd, to użytkownik nie będzie myślał, że nadal się ładuje